TYPO3.Neos/Resources/Public/JavaScript/Content/Application.js

```diff
@@ -324,6 +324,48 @@ function(
 			}
 		},
 
+		/**
+		 * Triggers reload of stylesheets if the given htmlDom (that is loaded via AJAX) contains different stylesheets links than the currently loaded page
+		 */
+		_loadNewStylesheets: function($htmlDom) {
+			if (window.navigator.userAgent.indexOf('MSIE ') > -1) {
```

Comment:
You removed the comment `We cannot do this stylesheet change on IE < 11 (the user agent string changed with 11, so everything with MSIE in is below 11)` but I think that was helpful (mainly the fact that this will _not_ exclude MSIE 11+)